### DISCUSSION TITLE
can the Riverpod be used in production? your opinion

### DISCUSSION THREAD

**Discussion by FetFrumos** [rrousselGit/riverpod#527] [General]
I started making a new app. chose state management for a long time. I used to use Bloc now settled on Riverpod.
In the process, questions arose - now I doubt my choice a little. I'm interested in the opinion of people who have been working with this plugin for a long time. are you happy with your choice?
is it ready for production?

**Comment by rrousselGit** [comment]
Why are you doubting your choice?

**Comment by julienlebren** [comment]
I am currently developing an app where I use riverpod for state management.
I also had doubts in the beginning because it does not look like the other existing solutions, which may be confusing on the first use cases.
But finally, I persevered and I'm happy with my choice, the way it works solves a lot of problems that other state management solutions don't solve.
Also, that's an active package with regular good improvements, which is an important point to me.
Feel free to asks questions about your doubts.
